clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/core/AccountHealthIndicator.java

```diff
@@ -45,6 +49,11 @@
   @Nonnull private Health health = new Health.Builder().up().build();
   @Nonnull private final AtomicLong unhealthyAccounts = new AtomicLong(0);
 
+  @Value("${clouddriver.account-threads:-1}")
+  private int threads;
+
+  private ForkJoinPool customThreadPool;
```

Comment:
Given that this health check always returns "up" and the change of `ThreadPoolTaskScheduler` will unblock parallel `@Scheduled` executions, probably creating another thread pool in clouddriver doesn't add much value.